pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```
@@ -0,0 +1,29 @@
+{ ffmpeg_4, ffmpeg-full, fetchFromGitHub, lib }:
+(ffmpeg-full.override { ffmpeg = ffmpeg_4; }).overrideAttrs (old: rec {
+  name = "jellyfin-ffmpeg";
+  version = "4.4.1-4";
+
+  src = fetchFromGitHub {
+    owner = "jellyfin";
+    repo = "jellyfin-ffmpeg";
+    rev = "v${version}";
+    sha256 = "0y7iskamlx30f0zknbscpi308y685nbnbf5gr9cj1znr5dlfb0bn";
+  };
+
+  prePatch = ''
+    for file in $(cat debian/patches/series); do
+      patch -p1 < debian/patches/$file
+    done
+
+    ${old.prePatch}
```

```suggestion
  postPatch = ''
    for file in $(cat debian/patches/series); do
      patch -p1 < debian/patches/$file
    done

    ${old.postPatch}
```

I think we should mess with the files after applying patches.